contracts/Gauge.sol

```diff
@@ -120,30 +117,6 @@ contract Gauge is BaseGauge, ERC20Upgradeable, IGauge {
             _ve,
             _veYfiRewardPool
         );
-        boostingFactor = 100;
-    }
-
-    /**
-    @notice Set the veYFI token address.
-    @param _veToken the new address of the veYFI token
-    */
-    function setVe(address _veToken) external onlyOwner {
```

Comment:
so in case of a gauge being bad or deprecated we just migrate to new contracts instead of changing ve right? assume there would not be any rewards incoming on this bad gauge

Reply:
makes sense